### PR TITLE
Adding changes for supporting remote script location

### PR DESCRIPTION
**Pull request by tarrygo**
These changes are for supporting S3 scripts for spark commands.

### CODE REVIEW

File: tests/test_command.py [hunk 1]
```diff
@@ -428,10 +428,27 @@ def test_submit_notebook_with_program(self):
             qds.main()
 
     def test_submit_script_location_aws(self):
```
Comment (sourabh912) [inline tests/test_command.py]: @tarrygo : Please add a test case for unsupported file extension.

Comment (sourabh912) [inline tests/test_command.py]: @tarrygo : Also please add tests for aws scala, R and sql just like we have tests for local script location.

**Comment by tarrygo** [comment]
Added testcase to check for language and failure case.

**Comment by sourabh912** [comment]
Merged into unreleased. Commit Link: https://github.com/qubole/qds-sdk-py/commit/cb29fd258faf29a851b7d1c0b4abd3a1f6076b92

**Pull request closed without merging**